assert.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# Assert the changes made by this action.
+#
+
+set -eu -o pipefail
```

Comment:
You can short this if you want.

```suggestion
set -euo pipefail
```

Reply:
I think the `-o` is more descriptive, that construction was intentional.